hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -36,354 +31,271 @@ import java.time.LocalDateTime
  * look in JSON format see test class [AttributeSearchRequestJsonTest].
  */
 class AttributeSearchIntegrationTest : AbstractSearchIntegrationTest() {
-  private val prisonerBuilders = mutableListOf<PrisonerBuilder>()
-
-  // The super class refreshes the index before this test class runs, but we don't want it to load any prisoners yet
-  override fun loadPrisonerData() {}
-
-  @BeforeEach
-  fun `clear search repository`() {
-    prisonerBuilders.onEach { prisonerRepository.delete(it.prisonerNumber) }.clear()
-    waitForPrisonerLoading(expectedCount = 0)
+  private val now = LocalDateTime.now()
+
+  // Create 3 prisoners with predictable data we can search for
+  val prisoners = listOf(
+    PrisonerBuilder(
+      prisonerNumber = "P1",
+      firstName = "John1",
+      dateOfBirth = "1990-01-01",
+      heightCentimetres = 181,
+      recall = false,
+      currentIncentive = IncentiveLevelBuilder(levelDescription = "I1", dateTime = now.minusDays(2).minusHours(1)),
+      alertCodes = listOf("AT1" to "AC1"),
+      profileInformation = ProfileInformationBuilder(youthOffender = true),
+    ),
```

Comment:
Much preferred the old tests when it was obvious which piece of test data caused the result for each test. Now you have to refer back to these prisoners to understand why each test works. But by loading the test data only once we've cut 90% off the test running time so it's probably worth it.